product_input_interface/input_interface.py

```diff
@@ -0,0 +1,82 @@
+from product_input_interface.allowed_products import WarehouseProducts
+
+
+class InputInterface:
+
+    def __init__(self):
+        self.WAREHOUSE = WarehouseProducts()
```

Comment:
`self.WAREHOUSE = WarehouseProducts()` it seems to be a little confusing - so is it about warehouse or warehouse products? I would rename it like this:
```suggestion
        self.WAREHOUSE_PRODUCTS = WarehouseProducts()
```